Index: main/app/composite/master-detail/master.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- main/app/composite/master-detail/master.js	(revision 52d8aa9c192bb1e0ea84db757a50d19e518f9623)
+++ main/app/composite/master-detail/master.js	(revision )
@@ -467,8 +467,11 @@
                      */
                     $scope.itMasterDetailControl.getSelectedItems = function(){
                         if(typeof $scope.gridApi !== 'undefined' ) {
-                            if (typeof $scope.gridApi.selection.getSelectedRows() === 'function') {
+                            if (typeof $scope.gridApi.selection.getSelectedRows === 'function') {
                                 return $scope.gridApi.selection.getSelectedRows();
+                            }
+                            else  {
+                                return [];
                             }
                         } else  {
                             return [];
\ No newline at end of file
